### PR TITLE
Add note about adding the OpenSSL libraries to LDFLAGS when building

### PR DESCRIPTION
**Pull request by torarnv**

### CODE REVIEW

**Comment by torarnv** [comment]
cc @mattrobenolt

File: docs/internal/environment.rst [hunk 1]
```diff
@@ -78,6 +78,10 @@ and some extra pieces that hold the development environment together::
 
     make develop
 
+.. note:: Some libraries depend on OpenSSL. Install via ``brew install openssl``, and make
+  sure to ``export LDFLAGS=-L/usr/local/opt/openssl/lib`` while building, so that the
```
Comment (dcramer) [inline docs/internal/environment.rst]: the indentation here might need lined up with the start of the next line

Comment (torarnv) [inline docs/internal/environment.rst]: Hmm, I used the same indentation as the notes above and below it I think...

Comment (dcramer) [inline docs/internal/environment.rst]: ah ok then just my RST paranoia probably

**Pull request merged**